cmd/root.go

```diff
@@ -287,7 +294,12 @@ func writeStartupMessage(c *cobra.Command, sched time.Time, filtering string) {
 		startupLog.Info("Scheduling first run: " + sched.Format("2006-01-02 15:04:05 -0700 MST"))
 		startupLog.Info("Note that the first check will be performed in " + until)
 	} else {
-		startupLog.Info("Running a one time update.")
+		startupLog.Info("Periodic runs are not enabled.")
+	}
```

Comment:
This is a bit weird when just starting with `--run-once`, as it will be followed by said update.

```suggestion
		if runOnce, _ := c.PersistentFlags().GetBool("run-once"); runOnce {
			startupLog.Info("Running a one time update.")
        	} else {
			startupLog.Info("Periodic runs are not enabled.")
		}
	}
```

It should probably be `else if` etc. but that's not supported by GitHub suggestions...

Reply:
It made sense to me, but you could be right that not to anyone else so better to be more helpful. Added the else if version.